wicket-core/src/test/java/org/apache/wicket/core/request/mapper/MountedMapperTest.java

```diff
@@ -714,7 +714,7 @@ public boolean isNewPageInstance()
 	@Test
 	public void placeholderEncode4()
 	{
-		PageProvider provider = new PageProvider(new MockPage())
+		PageProvider provider = new PageProvider(MockPage.class)
```

Comment:
Not the same and the change is needed. This test follows up by changing the page provider which will have no impact on the provider that already holds an instance.